src/main/java/com/google/api/codegen/util/java/JavaTypeTable.java

```diff
@@ -81,6 +83,12 @@ public TypeName getTypeName(String fullName) {
   }
 
   @Override
+  public TypeName getTypeNameFromShortName(String shortName) {
```

Comment:
It is not clear from the name what this does. It should probably be something like 'getTypeNameInImplicitPackage'.